frontend/playwright.config.ts

```diff
@@ -102,9 +104,17 @@ const config: PlaywrightTestConfig = {
 
   /* Configure projects for major browsers */
   projects: [
+    {
+      name: "preSetup",
```

Comment:
preSetup will conditionally fetch access token if it detects it's running against an rdev